packages/insert/src/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @description Inserts into the input text a string at specified position
+ * @param {String} text
+ * @param {String} textToInsert
+ * @param {Number} insertPosition
+ *
+ * @example
+ * insert('foo ba'); // returns  'foo ba'
+ * insert('foo ba', "r"); // returns  'foo bar'
+ * insert('foo ba', "r", 6); // returns  'foo bar'
+ * insert('foo ba', "r", 100); // returns  'foo bar'
+ * insert('foo ba', "r", -100); // returns  'foo bar'
+ * insert('oo bar', "", 0); // returns  'foo bar'
```

Comment:
Should the return value of `insert('oo bar', "", 0);` return `'foo bar'` or `'oo bar'`

Reply:
My bad. You are right. 

I'll change it to ` * insert('oo bar', "f", 0); // returns  'foo bar'`

Thank you!